spec/requests/api/policies_spec.rb

```diff
@@ -352,4 +352,65 @@ def test_policy_profile_query(object, object_policy_profiles_url)
       test_policy_profile_query(template, template_policy_profiles_url)
     end
   end
+
+  context "Policy CUD actions" do
+    let(:action) { FactoryGirl.create(:miq_action) }
+    let(:condition) { FactoryGirl.create(:condition) }
+    let(:event) { FactoryGirl.create(:miq_event_definition) }
+    let(:miq_policy) { FactoryGirl.create(:miq_policy) }
+
+    let(:sample_policy) do
+      {
+        "description"    => "sample policy",
+        "name"           => "sample policy",
+        "towhat"         => "ManageIQ::Providers::Redhat::InfraManager",
+        'conditions_ids' => [condition.id],
+        'action_id'      => action.id,
+        'event_id'       => event.id
+      }
+    end
+
+    it "creates new policy" do
+      api_basic_authorize collection_action_identifier(:policies, :create)
+      run_post(policies_url, sample_policy)
+      policy_id = response.parsed_body["results"].first["id"]
+
+      expect(MiqPolicy.exists?(policy_id)).to be_truthy
+
+      expect(MiqPolicy.find(policy_id).conditions.count).to eq(1)
+
+      expect(MiqPolicy.find(policy_id).actions.count).to eq(1)
+
+      expect(MiqPolicy.find(policy_id).events.count).to eq(1)
+
```

Comment:
Maybe fetch it once do the expects on that (lotsa find() above).  Also, maybe remove the white space between each expect.